AmazonSQS/AmazonSQSExample/pom.xml

```diff
@@ -4,12 +4,12 @@
     <parent>
         <groupId>fish.payara.cloud.connectors</groupId>
         <artifactId>cloud-connectors</artifactId>
-        <version>0.8.0</version>
+        <version>0.9.0-SNAPSHOT</version>
```

Comment:
OK for versioning I think we should just increment the major version by 1.
So this would be the 1.0 release, followed by 1.1-SNAPSHOT as the next dev version.

Jakarta namespace, assuming it gets merged afterwards, would then be another major increment for all of the connectors, so the other connectors would be 1.x and this connector would be 2.x

Reply:
got your point, for now I upgrade the version of the SQS for 1.0.0 for the release. I'm going to generate a tag for it and then move to SNAPSHOT